CMake/External_Protobuf.cmake

```diff
@@ -1,43 +1,50 @@
-if (WIN32)
-  # Build option for windows not yet generated
-  message( FATAL_ERROR "Protobuf on windows not yet supported" )
-endif()
 
-# Check that python and protobuf versions are compatible
-if(fletch_BUILD_WITH_PYTHON AND fletch_ENABLE_Protobuf)
-  # Note the python protobuf wrapper is not installed here.
-  # Instead it must be installed via `pip install protobuf`
-  if (${Protobuf_version} LESS 3.0 AND fletch_PYTHON_MAJOR_VERSION MATCHES "^3.*")
-    message(ERROR " Must use Protobuf >= 3.x with Python 3.x")
+if (Protobuf_SELECT_VERSION STREQUAL "3.9.0")
+  ExternalProject_Add(Protobuf
+#    PREFIX protobuf
+    URL ${Protobuf_url}
+    URL_MD5 ${Protobuf_md5}
+    SOURCE_SUBDIR ./cmake
+    ${COMMON_EP_ARGS}
+    ${COMMON_CMAKE_EP_ARGS}
+#    UPDATE_COMMAND
+#        COMMAND ${CMAKE_COMMAND} -DPULSE_IL2CPP_PATCH=${PULSE_IL2CPP_PATCH} -Dprotobuf_source=${protobuf_SRC} -Dprotobuf_patch=${protobuf_Patch} -P ${protobuf_Patch}/Patch.cmake
```

Comment:
You can take this patch out. It was put in to work with the Unity IL2CPP compiler that converts C# to C++.
The IL2CPP compiler can be a bit too aggressive, so this patch was created
By the time you are supporting kwiver/fletch into C# and Unity, this will probably have changed
Just let me know then

Reply:
Thanks ... what are you thoughts about making protobuf3 the default? It certainly works building/testing kwiver on all platforms.